src/components/cards/CreatePool/CreateActions.vue

```diff
@@ -70,10 +70,8 @@ const {
 } = usePoolCreation();
 const { networkSlug } = useNetwork();
 
-/**
- * COMPUTED
- */
 const actions = ref<TransactionActionInfo[]>([
+  ...tokenApprovalActions.value,
```

Comment:
I'm pretty sure this won't work because I just changed it from this to fix a bug. This is a ref not a computed property, so it's only set on component setup. If the approvals have not been fetched via the tokens.provider allowances query then it returns an incorrect tokenApprovalActions array. That is why below in the `onBeforeMount` hook I had to use the explicit fetch method.

Reply:
You are right

Reply:
Rolled back